tests/test_utils_datatypes.py

```diff
@@ -279,8 +282,10 @@ def test_set(self):
         self.assertIn(set("bar"), d)
 
         # supplied sequence is a set, so checking for list (non)inclusion fails
-        self.assertRaises(TypeError, (0, 1, 2) in d)
```

Comment:
hm, this wasn't working before, right?

Reply:
I'm not sure what was happening before here

Reply:
It seems it was just a wrong test, checking that tuple should raise an error, while it wasn't raising it

Reply:
> checking that tuple should raise an error

But the comment talks about a list, so I also don't know what was this supposed to test.